Documentation/getting-started-android.md

```diff
@@ -235,11 +239,11 @@ You can download these files from [nnsuite testcases repository](https://github.
 ```
 
 To check the testcases, run the build script with an option ```--run_test=yes```.
-You can find the result in ```$NNSTREAMER_ROOT/android_lib```.
+You can find the result in ```$ML_API_ROOT/android_lib```.
 
 ```bash
-$ cd $NNSTREAMER_ROOT
-$ bash ./api/android/build-android-lib.sh --run_test=yes
+$ cd $ML_API_ROOT
+$ bash ./java/android/build-android-lib.sh --run_test=yes
```

Comment:
fix script to `build-nnstreamer-android-lib.sh`

Reply:
Updated, Thanks!